client/my-sites/themes/theme-options.js

```diff
@@ -45,7 +46,7 @@ const buttonOptions = {
 		// We don't know where support docs for a given theme on a self-hosted WP install are,
 		// and free themes don't have support docs.
 		isHidden: ( site, theme ) => ( site && site.jetpack ) || ! Helper.isPremium( theme )
-	},
+	}
```

Comment:
http://eslint.org/docs/rules/comma-dangle.html - you are right, it's no longer an issue and `comma-dangle` is disabled. There is a big chance that Babel takes care of it. Old habits die hard :D